fintrack/transactions/tasks.py

```diff
@@ -0,0 +1,40 @@
+from celery import shared_task
+from decouple import config
+from django.core import mail
+from django.utils import timezone
+
+from .models import Transaction
+
+@shared_task
+def send_transaction_emails():         
+    transactions = Transaction.objects.filter(
+        date=timezone.now().date()
+    )
+    user_emails = transactions.values_list(
+        'transaction_from_account__holder__email',
+        flat=True
+    ).distinct()
```

Comment:
wouldn't it be much easier to loop through users and then fetch all of the transaction osf the current user?